pages/Cart/index.jsx

```diff
@@ -94,16 +96,47 @@ class Cart extends Component {
           {messages.length > 0 && <MessageBar messages={messages} />}
           {cartItems.length > 0 && (
             <Fragment>
-              <CardList>
-                {cartItems.map(cartItem => (
-                  <Item
-                    key={cartItem.id}
-                    item={cartItem}
-                    togglePaymentBar={this.togglePaymentBar}
-                  />
-                ))}
-                <CouponField onToggleFocus={this.togglePaymentBar} />
-              </CardList>
+              <Portal name={portals.CART_ITEM_LIST_BEFORE} />
+              <Portal name={portals.CART_ITEM_LIST}>
+                <CardList>
+                  {cartItems.map(cartItem => (
+                    <Fragment key={cartItem.id}>
+                      <Portal
+                        name={portals.CART_ITEM_BEFORE}
+                        props={{
```

Comment:
Please store that object in a variable before instead of creating a new object all the time.